presto-parser/src/main/java/io/prestosql/sql/testing/TreeAssertions.java

```diff
@@ -63,9 +63,8 @@ private static Statement parseFormatted(SqlParser sqlParser, ParsingOptions pars
             return sqlParser.createStatement(sql, parsingOptions);
```

Comment:
Can you please add to commit messages of all commits:

```
Extracted from: https://github.com/prestodb/presto/pull/11561
```

Reply:
I don't think it's needed for a one line fix